README.md

```diff
@@ -6,12 +6,13 @@ StopModReposts
 ![OpenCollective](https://opencollective.com/stopmodreposts/tiers/badge.svg)
 ![GitHub checks](https://badgen.net/github/checks/StopModReposts/Illegal-Mod-Sites)
 
-> 👋 We're searching for Project Maintainers. If you're interested, send an email to help[at]stopmodreposts.org
+> 👋 We're searching for Project Maintainers. If you're interested, send an email to help@stopmodreposts.org
```

Comment:
Pretty sure [at] was to prevent some bots from spamming the email.

Reply:
> Pretty sure [at] was to prevent some bots from spamming the email.

you could just use an [emailencoder](http://www.wbwip.com/wbw/emailencoder.html) and that should stop the bots

Reply:
Using [at] is like dada said to prevent common email spam while still preserving readability compared to an encoded email address